buildenv/jenkins/testJobTemplate

```diff
@@ -39,10 +39,12 @@ if (!binding.hasVariable('JCK_GIT_REPO_PREFIX')) JCK_GIT_REPO_PREFIX = ""
 if (!binding.hasVariable('SSH_AGENT_CREDENTIAL')) SSH_AGENT_CREDENTIAL = ""
 if (!binding.hasVariable('KEEP_REPORTDIR')) KEEP_REPORTDIR = "true"
 if (!binding.hasVariable('AUTO_DETECT')) AUTO_DETECT = true
+if (!binding.hasVariable('USE_JRE')) USE_JRE = false
 if (!binding.hasVariable('TEST_JOB_NAME')) TEST_JOB_NAME = ""
 if (!binding.hasVariable('TARGET')) TARGET = ""
 if (!binding.hasVariable('BUILD_LIST')) BUILD_LIST = ""
 if (!binding.hasVariable('SDK_RESOURCE')) SDK_RESOURCE = "upstream"
+if (!binding.hasVariable('CUSTOMIZED_SDK_URL_CREDENTIAL_ID')) CUSTOMIZED_SDK_URL_CREDENTIAL_ID = ""
```

Comment:
The value needs to be set in https://github.com/adoptium/aqa-tests/blob/ae17f0ce0258d17747cda13404eb6c012f7deb36/buildenv/jenkins/testJobTemplate#L312 for it to take effect.

Reply:
not sure I understand what I need to do... but confirming that this currently does not behave how I want it (where I want to set the value in Test_Job_Auto_Gen "Build with Parameters" view).  How to accomplish that?

Reply:
Instead of setting an empty string, [Line 312](https://github.com/adoptium/aqa-tests/blob/ae17f0ce0258d17747cda13404eb6c012f7deb36/buildenv/jenkins/testJobTemplate#L312) has to take in CUSTOMIZED_SDK_URL_CREDENTIAL_ID. For example:
```
stringParam('CUSTOMIZED_SDK_URL_CREDENTIAL_ID', CUSTOMIZED_SDK_URL_CREDENTIAL_ID, "Only use this if you are pulling an JDK from a site that needs credential")
```

Reply:
thanks, and I verified with temurin-compliance/Test_Job_Auto_Gen/59